recipes/itk-dev/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "4.11.0" %}
+
+package:
+  name: itk-dev
+  version: "{{version}}"
+
+source:
+  url: https://github.com/InsightSoftwareConsortium/ITK/archive/v{{version}}.tar.gz
+
+
+build:
+    number: 0
+
+requirements:
+  build:
+    - cmake    >=3.3
+    - ninja
+    - libtiff  4.0.*
+    - libpng   >=1.6.27,<1.7
+    - jpeg     9*
+    - zlib     1.2.*
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+  run:
+    - libtiff  4.0.*
+    - libpng   >=1.6.27,<1.7
+    - jpeg     9*
+    - zlib     1.2.*
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+
+test:
+  commands:
+    - echo "Testing..."
```

Comment:
I was thinking of configuring minimal CMake project which uses ITK. But we'll come up with a couple things, not that we have enough time in the build.